proto/tendermint/abci/types.proto

```diff
@@ -212,6 +211,12 @@ message ResponseCheckTx {
   repeated Event events     = 7
       [(gogoproto.nullable) = false, (gogoproto.jsontag) = "events,omitempty"];
   string codespace = 8;
+  string sender    = 9;
+  int64  priority  = 10;
+
+  // mempool_error is set by Tendermint.
+  // ABCI applictions creating a ResponseCheckTX should not set mempool_error.
+  string mempool_error = 11;
```

Comment:
We removed this for v0.36 but I think it's fine if we just leave it for now since it's also in v0.35

Reply:
Yes, the way I'm doing this PR is just dropping in the entire mempool from 0.35.x into 0.34.x, modulo any API changes necessary.